requirements.txt

```diff
@@ -1,4 +1,5 @@
 matplotlib
-numpy>=1.15.0
+numpy>=1.14.2
+pytest
```

Comment:
Ahh, I thought this might come up eventually. Can you please remove the `pytest` dependency?
I'm going to include instructions for contributing (including how to setup your dev environment) [here]()

Reply:
Resolved, sorry about that.